src/main/java/Lovie.java

```diff
@@ -0,0 +1,214 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Lovie {
+    public static void main(String[] args) {
+        Scanner inputGetter = new Scanner(System.in);
+        String input;
+        ArrayList<Task> tasksList = new ArrayList<Task>();
+
+        String LOGO = "██╗░░░░░░█████╗░██╗░░░██╗██╗███████╗ \n" +
+                "██║░░░░░██╔══██╗██║░░░██║██║██╔════╝ \n" +
+                "██║░░░░░██║░░██║╚██╗░██╔╝██║█████╗░░ \n" +
+                "██║░░░░░██║░░██║░╚████╔╝░██║██╔══╝░░ \n" +
+                "███████╗╚█████╔╝░░╚██╔╝░░██║███████╗ \n" +
+                "╚══════╝░╚════╝░░░░╚═╝░░░╚═╝╚══════╝ \n";
+        print(LOGO + "Hey hey! My name is Lovie! How can I help you today?");
```

Comment:
Try not to use magic literals e.g. magic numbers and magic string, consider giving it a meaning by creating a constant and using the constant in your code.